src/model/transaction/updateEntityDataInContentState.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule updateEntityDataInContentState
```

Comment:
Module name should match exported function name. (Lint will complain about this internally.)